### PR TITLE
Blog: fix typos in APISIX blog

### PR DESCRIPTION
**Pull request by yzeng25**
Found some typos as I was going through the blog again.

1. Change "OpenTelemetry" to "OpenTelemetry" in the topic.
2. Change "affectting" to "affecting".
3. Change "OpenTelemetery" to "OpenTelemetry".
4. Change "docekr-compose.yaml" to "docker-compose.yaml"

### CODE REVIEW

**Comment by yzeng25** [comment]
Hi @chalin @SergeyKanzhelev , sorry to bother you guys again, could you take a look on this pull request when available, thanks a lot!

**Review by chalin — APPROVED** [review]
Great! Thanks for catching those through another careful review of your post. ✨

File: content/en/blog/2022/apisix.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
-title: Apache APISIX Integrates with OpenTelementry to Collect Tracing Data
+title: Apache APISIX Integrates with OpenTelemetry to Collect Tracing Data
 linkTitle: Apache APISIX-Opentelemetry Integration
 date: 2022-03-26
 author: Haochao Zhuang, Fei Han
+canonical_url: https://apisix.apache.org/blog/2022/02/28/apisix-integration-opentelemetry-plugin/
```
Comment (chalin) [inline content/en/blog/2022/apisix.md:6]: I'm unsure what our policy will be about canonical URLs for regular blog posts (btw, this doesn't set the page's canonical URL yet). I'll bring this up during our next Comms/website meeting.

Comment (yzeng25) [reply content/en/blog/2022/apisix.md:6]: Is it required to add `[original post]: {{% param canonical_url %}}` as well to complete the canonical URL?

Comment (chalin) [reply content/en/blog/2022/apisix.md:6]: You can certainly add that to the end of your post, similarly to what was done for https://opentelemetry.io/blog/2022/troubleshooting-nodejs/ (see the end of the page). That is:

> _A version of this article was [originally posted](#) on the Apache APISIX blog._

Using `{{% param canonical_url %}}` for the link definition. See https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/blog/2022/troubleshooting-nodejs.md for an example.

Comment (yzeng25) [reply content/en/blog/2022/apisix.md:6]: Ah I get it now, thanks @chalin!

**Pull request merged**